src/cli/dev.test.ts

```diff
@@ -118,7 +118,7 @@ describe("dev command", () => {
     const program = new Command();
     devCommand(program);
 
-    await program.parseAsync(["node", "test", "dev"]);
+    await program.parseAsync(["npx", "pepr", "dev"]);
```

Comment:
```suggestion
    await program.parseAsync(["dev"], { from: "user" });
```
Consistency with other tests we've written, though it's functionally equivalent